docs/FAQ.rst

```diff
@@ -201,12 +201,16 @@ Run ``lgb.unloader(wipe = TRUE)`` in the R console, and recreate the LightGBM da
 Due to the pointers, choosing to not wipe variables will not fix the error.
 This is a known issue: `Microsoft/LightGBM#698 <https://github.com/microsoft/LightGBM/issues/698>`__.
 
-2. I used ``setinfo``, tried to print my ``lgb.Dataset``, and now the R console froze!
---------------------------------------------------------------------------------------
+2. I used ``setinfo()``, tried to print my ``lgb.Dataset``, and now the R console froze!
+----------------------------------------------------------------------------------------
 
 Avoid printing the ``lgb.Dataset`` after using ``setinfo``.
 This is a known bug: `Microsoft/LightGBM#539 <https://github.com/microsoft/LightGBM/issues/539>`__.
 
+3. `error in data.table::data.table()...argument 2 is NULL`
```

Comment:
Underscore to fix formatting.

![image](https://user-images.githubusercontent.com/25141164/75875958-cfb88a00-5e25-11ea-98ee-2de1ea80f113.png)

Reply:
ah! I knew I'd miss one

Reply:
![image](https://user-images.githubusercontent.com/7608904/75901358-01b8f480-5e04-11ea-9cd2-15fc8fe78dcf.png)

ok looks better now